web/src/rest.js

```diff
@@ -25,7 +24,8 @@ const publishRest = axios.create({ baseURL: publishApiRoot });
 Object.assign(publishRest, {
   assetDownloadURI(asset) {
     const { uuid, version: { version, dandiset: { identifier } } } = asset;
-    return `${publishRest.defaults.baseURL}/dandisets/${identifier}/versions/${version}/assets/${uuid}/download`;
+    const baseURL = publishRest.defaults.baseURL.endsWith('/') ? publishRest.defaults.baseURL.slice(0, -1) : publishRest.defaults.baseURL;
```

Comment:
Isn't the appropriate place for this logic up on line 5, so `publishApiRoot` always has a consistent format?

Reply:
There is no function in js world to join urls like urllib.parse.urljoin in python?

Reply:
Good point, I'll fix it